reference/manual-upgrade.md

```diff
@@ -0,0 +1,391 @@
+## Manual Upgrade
+
+Upgrading Mayastor from a version prior to 2.0.0 needs manual intervention. Follow the steps given below.
```

Comment:
A legacy installation of Mayastor (1.0.5 and below) cannot be seamlessly upgraded and needs manual intervention.
Follow the below steps if you wish to upgrade from Mayastor 1.0.x to Mayastor 2.1.0 and above.